Chaincase.Android/Chaincase.Android.csproj

```diff
@@ -67,6 +66,14 @@
       <Version>2.4.1</Version>
     </PackageReference>
   </ItemGroup>
+  <PropertyGroup Condition="'$(Configuration)' == 'Release' ">
+    <DefineConstants>LIVESHARP_DISABLE</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(Configuration)' != 'Release' ">
+    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="3.1.8" />
+    <PackageReference Include="LiveSharp" Version="1.6.33" />
+  </ItemGroup>
```

Comment:
whoa does this run a client/server arch just for the app in debug mode?

Reply:
Livesharp is a beast indeed